app/src/main/java/com/github/se/eventradar/ui/chat/Chat.kt

```diff
@@ -61,24 +59,19 @@ import com.github.se.eventradar.viewmodel.ChatViewModel
 import java.time.LocalDateTime
 
 @Composable
-fun ChatScreen(viewModel: ChatViewModel = hiltViewModel(), navigationActions: NavigationActions) {
+fun ChatScreen(viewModel: ChatViewModel, navigationActions: NavigationActions) {
   val uiState by viewModel.uiState.collectAsState()
-  // Note the opponentId should no longer be in the uiState but be accessible with
-  // viewModel.opponentId
-  // TO DO: Implement get messages between two users in VM
-  //    viewModel.getMessages(senderId, opponentId)
 
-  val context = LocalContext.current // only needed until message send is implemented
+  viewModel.initOpponent()
+  viewModel.getMessages()
```

Comment:
The functions above are already called when the viewModel is created. View ChatViewModel's `init` block.